lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -2061,12 +2061,17 @@ def main():
                 # Register the vm into the system:
                 changed = True
                 vm_service = vms_service.vm_service(vm.id)
+                sp_id = storage_domain_service.get().data_centers[0].id
+                dc_name = connection.system_service().data_centers_service(). \
+                    data_center_service(sp_id).get()._name
+                logging.info(dc_name)
                 vm_service.register(
                     allow_partial_import=module.params['allow_partial_import'],
                     cluster=otypes.Cluster(
                         name=module.params['cluster']
                     ) if module.params['cluster'] else None,
-                    vnic_profile_mappings=_get_vnic_profile_mappings(module)
+                    vnic_profile_mappings=_get_vnic_profile_mappings(module,
```

Comment:
Ansible has allowed 160chars per line, I think it's OK to have this on single line.

Reply:
done